### PR TITLE
Added Mac compatibilty for Android scripts

### PR DESCRIPTION
**Pull request by petermcneil**

### CODE REVIEW

**Review by EraYaN — APPROVED** [review]
LGTM

File: .gitignore [hunk 1]
```diff
@@ -28,7 +28,6 @@ project.xcworkspace
 
 # Android/IntelliJ
 #
-build/
```
Comment (LucaBernstein) [inline .gitignore:31]: Why did you remove this line? Do we really want the build/ folder to be committed?

Comment (petermcneil) [reply .gitignore:31]: It's dupilcated from the XCode section - line 7. Happy to add it back to make it explicit for Android.

Comment (LucaBernstein) [reply .gitignore:31]: Maybe moving it to a "commons" section would also help ;) but you are right, it's still in there :)

**Review by DrPandemic — APPROVED** [review]
Not tested on Mac, but still build on Linux

**Comment by EraYaN** [comment]
Please rebase this PR after #16 was merged.

**Comment by petermcneil** [comment]
Rebased and reordered files
